### DISCUSSION TITLE
Bugfix update: Version 0.4.1

### DISCUSSION THREAD

**Discussion by kaixxx** [kaixxx/noScribe#31] [Announcements]
Version 0.4 had a bug which caused problems in combination with NVIDIA graphics cards under Windows. Vers. 0.4.1 fixes this. If you did not experience this bug, you don't have to update. 

Discuss your experiences with Vers. 0.4.1 here, positive or negative.

**Comment by robertaust** [comment]
<img width="870" alt="Bildschirmfoto 2023-10-13 um 16 28 47" src="https://github.com/kaixxx/noScribe/assets/149690243/eb552cd9-4256-443b-8ef2-5e7c54bbfd28">
It seems that my mac is blocking pyannote? If somebody can help me, would be great to use noscribe
best, r

**Reply by gernophil** [reply]
Hey @robertaust,
welche Version nutzt Du denn? Mich wunderts, dass das in 'Applications/noScribe.app/Contents/Frameworks/models/' geschrieben werden soll. Das war in einer alten Version so. In den neuen Versionen sollte das eigentlich in ein temporäres Verzeichnis geschrieben werden.

**Reply by robertaust** [reply]
Hey Gernophil, tatsächlich läuft es in der neuen Version! Sorry für die Aufregung :) ist soweit alles iO
Und Danke für die schnelle Antwort!

**Reply by gernophil** [reply]
Welche Version nutzt du denn und welchen Mac hast du :)? Mich interessiert vor allem der out of memory error, der bei neueren Versionen je nach Arbeitsspeicher auftritt.

**Reply by robertaust** [reply]
MacBook Air M1, 16 GB Sonoma

**Reply by gernophil** [reply]
And what noScribe version did you download?

**Reply by robertaust** [reply]
die letzte. aber wie gesagt, funktioniert alles bestens :)

**Comment by BabyFnord** [comment]
@kaixxx Die 0.4.1 für WIN benötigt leider wieder Admin-Rechte zur Installation. Bei der 0.3 war das noch anders (auf meine damalige Nachfrage). Könntest du das bitte wieder ändern? Danke :)

**Comment by kaixxx** [comment]
@BabyFnord Unter Win brauchen Installationen eigentlich immer Adminrechte. Ich konnte das bei Vers. 0.3 nur ändern, weil noScribe aus anderen Gründen sowieso im User-Verzeichnis installiert werden musste. Das wiederum wurde von der IT-Abteilung einer Universität harsch kritisiert, die sich u.a. deshalb weigerte, noScribe auf Dienstrechnern zuzulassen. 
Inzwischen installiert sich noScribe ganz normal in den Programmordner und das geht leider nur mit Adminrechten. 
(Einzig denkbarer workaround, ungetestet: noScribe auf einen anderen PC isntallieren, den ganzen Installtionsordner dann via USB-Stick auf den Rechner ohne Adminrechte transferieren. Könnte funktionieren...)

**Reply by BabyFnord** [reply]
Damit ist für einfache Studis ohne Privileg eines Dienstrechners der Zugang zu potenten Rechnern in Computerpools verwehrt. Ich schätze mal frech, dass die Zahl ersterer größer als der Dienstrechnerbesitzer:innen ist. Andere Software geht ja auch ohne Admin-Rechte zu installieren (random Beispiele: Sublime Text, IrfanView, …) … aber wenn IT-Abteilungen hier das letzte Wort haben dürfen, dann geht es hoffentlich doch mit o. g. workaround.

**Reply by kaixxx** [reply]
Ok, das ist ein Argument. Ich wusste gar nicht, dass es solche Computerpools noch gibt...
Mal schauen, ob ich bei der nächsten Version eine Wahlmöglichkeit einführen kann und Adminrechte nur bei Installation im Programmorder verlange. Ich kann aber nichts versprechen, das scripting des Installationstools ist eine Wissenschaft für sich.
